### PR TITLE
Feature/bds 938 background shapes tests

### PR DESCRIPTION
**Pull request by remydenton**
## Jira

http://vjira2:8080/browse/BDS-938

## Summary

Adds jest tests and manual testing steps for background shapes component

## Details

This is a low visibility component-- I mostly wanted to make sure I'm doing this right and ask a few questions before I add tests to a bunch of other components.

### For developers
- [x] Confirm that the new jest test coverage for the background shapes component is adequate
### For QA
- [ ] Perform manual testing of the steps described in packages/components/bolt-background-shapes/TESTING.md

### CODE REVIEW

File: packages/components/bolt-background-shapes/__tests__/background-shapes.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const { render } = require('@bolt/twig-renderer');
+const { readYamlFileSync } = require('@bolt/build-tools/utils/yaml');
+const { join } = require('path');
+const schema = readYamlFileSync(
+  join(__dirname, '../background-shapes.schema.yml'),
+);
+const { shapeGroup } = schema.properties;
+
+describe('background shapes', async () => {
+  shapeGroup.enum.forEach(async group => {
```
[the remark this answers is not in the document]
Comment (sghoweri) [reply packages/components/bolt-background-shapes/__tests__/background-shapes.js]: @remydenton Jest has an automatic (15 second?) timeout set for every test that's being run in order to make sure tests run quickly (and fwiw - this timeout get increased if an unusual test pops up that needs more time to run).

TLDR: you'd know if this was a performance issue 🙂

File: packages/components/bolt-background-shapes/TESTING.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Background shapes testing steps
```
[the remark this answers is not in the document]
Comment (sghoweri) [reply packages/components/bolt-background-shapes/TESTING.md]: @remydenton they can once #1050 get in. 

For example: 
https://feature-pattern-lab-refactor.boltdesignsystem.com/pattern-lab/?p=viewall-components-button

101e9414-2971-4334-8d83-d2c192c1fbed

File: packages/components/bolt-background-shapes/__tests__/background-shapes.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (remydenton) [reply packages/components/bolt-background-shapes/__tests__/background-shapes.js]: I was just thinking of a death-by-a-thousand-cuts scenario-- say, tests for each component taking 6 seconds instead of 3, multiplied by 20 components...

We can just cross that bridge when we come to it though.
